file-sharing-back/file-sharing-back-core/src/main/java/com/file/sharing/core/objects/Context.java

```diff
@@ -29,4 +31,10 @@
 	 */
 	StorageInfo getUserStorageInfo();
 
+	/**
+	 *
+	 * @return
+	 */
+	Map<Long, Long> getUserUsedAndTotalSpace();
```

Comment:
Don't think this is ment to be kept here... maybe it was just for testing purpouse

Reply:
The implementation for this one is flagged as TODO, so I'll leave it for now